bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/SecurityCapability.java

```diff
@@ -132,7 +136,7 @@ protected List<NAObject> updateReadings(SecurityApi api) {
                         eventBuffer.put(personId, event);
                     }
                 }
-                if (freshestEventTime == null || event.getTime().isAfter(freshestEventTime)) {
+                if (event.getTime().isAfter(freshestEventTime)) {
```

Comment:
Sure this is not resulting in an NPE? 

isAfter docs:
```
Parameters:
other the other date-time to compare to, not null
```

Reply:
Yes because I made changes so `freshestEventTime` can not be null.